superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -164,6 +164,16 @@ export function FiltersConfigModal({
     addFilter,
   );
 
+  // After this, it should be as if the modal was just opened fresh.
+  // Called when the modal is closed.
+  const resetForm = () => {
+    form.resetFields();
```

Comment:
I think we don't need this line (`form.resetFields();`) because it cause issue that described in previous PR...
Also form will be destroyed automatically, after modal closed


But other line of this function is good to have - to reset local state :)

Reply:
Line removed! Thanks!